### DISCUSSION TITLE
<title>Creating a zip file for direct download.

### DISCUSSION THREAD

**Discussion by RobijnenTaart** [community/community#125068] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

You have a Minecraft resource pack that you want to share with your team, and you want to make sure they can always download the latest version. The problem you encounter is that when you package the resource pack into a ZIP file, an additional map is created with the same name as the ZIP file, and within the map that contains your files. What you want to achieve is that when someone unzips the ZIP file, the files are immediately available without them having to open another card first. Because that is how Minecraft wants it.

https://github.com/SageMC-Development/Creative-ResourcePack/releases/tag/LATEST_VERSION

**Comment by nikhilsbuilds** [comment]
Try using 7zip, maybe it can compress the files directyl
